client-s3/src/main/kotlin/app/cash/backfila/client/s3/record/Utf8StringNewlineStrategy.kt

```diff
@@ -6,7 +6,7 @@ import okio.ByteString
 /**
  * Produces single string records separated by newlines.
  */
-class Utf8StringNewlineStrategy(
+open class Utf8StringNewlineStrategy(
```

Comment:
wouldn't composition work well for this instead of inheritance?

Reply:
it would yes! I kept the changes small because I didn't have a lot of time.